TicTacToe.js

```diff
@@ -0,0 +1,320 @@
+//Readline bloiler plate code
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+}
+
+let playerStatus = 1;
+rowOne = [' 1','|','2','|','3'];
+rowDashes = ['---','---','---'];
```

Comment:
Nice pre-emptive duplication removal! Some people might think it is silly to introduce named variables or constants for values that are only used once. Those people are wrong! :-) 

However, I have a minor critique: you have defined these as global variables; it'd be better to make them constants (`const rowOne = ...`) because that (a) declares your intention more clearly and (b) pre-empts bugs where you might accidentally change their values.